Configurations.md

```diff
@@ -1007,7 +1007,40 @@ use lorem;
 use sit;
 ```
 
-See also [`reorder_imported_names`](#reorder_imported_names).
+See also [`reorder_imported_names`](#reorder_imported_names), [`reorder_imports_in_group`](#reorder_imports_in_group).
+
+## `reorder_imports_in_group`
+
+Reorder import statements in group
```

Comment:
That sounds fine. Or we could be more specific by saying like "This option takes effect only when `reorder_imports` is set to `true`."